kedro-training/docs/01_prerequisites.md

```diff
@@ -1,24 +1,25 @@
 # Training pre-requisites
-This Kedro training materials assume the following pre-requisites for your technical understanding.
+The Kedro training materials assume a pre-requisite level of technical understanding.
 
-We assume understanding of these technologies and concepts during the training. In order to maximise your learning experience from the Kedro training, please review them before going through the training (there are external resource links for you to get a refresher on each topic).
+To optimise your experience and learn the most you can from the Kedro training, please review the following before your training session. We provide external resource links for each topic.
 
 - Intermediate Python knowledge
   - [Basic Python](https://docs.python.org/3/tutorial/)
```

Comment:
I would group it like this:

 - Introduction to Python
   - Functions, loops, conditional statements, IO operation
   - Common data structures: lists, dictionaries, tuples
 - Intermediate Python
   - Installing Python packages using `pip`
   - Dependency management with `requirements.txt`
   - Python modules (e.g how to use __init__.py and relative and absolute imports)
   - Some familiarity with Python data science libraries, especially `pandas`, `scikit-learn`, Jupyter Notebook, iPython)
   - Using a virtual environment (Anaconda, venv, pipenv)